src/GTP.cpp

```diff
@@ -282,7 +282,7 @@ bool GTP::execute(GameState & game, std::string xinput) {
     } else if (command.find("clear_board") == 0) {
         Training::clear_training();
         game.reset_game();
-        std::make_unique<UCTSearch>(game).swap(search);
+        search.reset(new UCTSearch(game));
```

Comment:
make_unique, never "new".

Reply:
Fixed, re-reading it appears reset(new ...) is a c++11 suggestion from before make_unique.